test/modern_bpf/test_suites/syscall_exit_suite/capset_x.cpp

```diff
@@ -0,0 +1,255 @@
+#include "../../event_class/event_class.h"
+
+#ifdef __NR_capset
+
+#include <sys/capability.h>
+
+#ifndef CAP_PERFMON
+	#define CAP_PERFMON 38
+#endif	
+
+#ifndef CAP_BPF
+	#define CAP_BPF 39
+#endif	
+
+#ifndef CAP_CHECKPOINT_RESTORE
+	#define CAP_CHECKPOINT_RESTORE 40
+#endif 
+
+uint64_t capabilities_to_scap(unsigned long caps)
```

Comment:
I guess they are copied from `driver/ppm_flag_helpers.h` which should not included into the test cases?

Reply:
Unfortunately yes, include them it's quite a mess since we should need to have also all other flag definitions that are not related to this helper, maybe I can add a comment, or I don't know if we have better ideas :thinking:

Reply:
A comment would help... typically there is the risk of becoming out-of-sync over the time. Another idea would be to split out definitions into separate header files... which also have some disadvantages. Not sure if the effort is worth for this single test case, likely to revisit if more test cases require helper definitions

Reply:
Yeah, that's the point, this seems the unique case in which we need the entire helper... I don't like the approach but probably it is the best way right now, I will add a comment and let's see if we need to split it in following steps :)